services/QuillLMS/client/app/bundles/Connect/components/questions/responseComponent.jsx

```diff
@@ -439,11 +441,11 @@ class ResponseComponent extends React.Component {
 
   renderRematchAllButton = () => {
     const { filters } = this.props
-    let disabled = filters.numberOfResponses > 1000
+
     if (this.props.admin) {
       const text = this.state.progress ? `${this.state.progress}%` : 'Rematch Responses';
 
-      return (<button className="button is-outlined is-danger" disabled={disabled} onClick={this.rematchAllResponses} style={{ float: 'right', }} type="button">{text}</button>);
+      return (<button className="button is-outlined is-danger" disabled={!this.state.enableRematchAllButton} onClick={this.rematchAllResponses} style={{ float: 'right', }} type="button">{text}</button>);
```

Comment:
so as part of our linters, we deconstruct state and prop properties so we'd want to make this:
```
const { filters, admin } = this.props;
const { progress, enableRematchAllButton } =  this.state;
```

Reply:
Ah, good call, will change. Ya'll are making this code so clean!